### ISSUE TITLE
Scan for songs stuck

### ISSUE THREAD

**Issue by BasConijn** [StarGazer1258/BeatDrop#29]
I am using the ID/Key structure, but it gets stuck at the last three files.

![afbeelding](https://user-images.githubusercontent.com/2544928/58763935-ae0e3780-8561-11e9-83c8-ea47b7fa1e5d.png)

**Comment by zxsdklfhskyrvk** [comment]
Yeah,  I guess I'm having this issue as well.  Any chance there is a log so we can identify WHY it is hanging?

**Comment by zxsdklfhskyrvk** [comment]
This is likely happening due to a song folder that has been modified or corrupted.  In my case, I modified a song folder's contents.

Try moving your songs out to another folder,  then move only some back in at a time to locate the offending song.
